tests/unit/packages/Teams/js/tests/actionTests.test.ts

```diff
@@ -0,0 +1,508 @@
+// Licensed under the MIT License.
```

Comment:
These are imported from what used to exist in the JS repo?

Reply:
Correct, with some changes related to this PR.